src/test-1.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/extensions */
+import { addTask, deleteTask } from './text-1';
+
+describe('Test Add function', () => {
+  test('Add one new task from input (length > 0)', () => {
+    global.document.getElementById('text').value = '555';
+    expect(addTask()).toBe('555');
+  });
+
+  test('Adding a empty input === undefined', () => {
+    global.document.getElementById('text').value = '';
+    expect(addTask()).toBe('');
+  });
+
+  test('should return correct input', () => {
+    global.document.getElementById('text').value = '123';
+    expect(addTask()).not.toBe('51515');
+  });
+});
+
+describe('Test Delete function', () => {
+  test('should delete task', () => {
+    global.document.getElementById('text').value = '555';
+    expect(deleteTask('555')).not.toBe('555');
+  });
```

Comment:
- [ ] I have concerns respecting this test. If I'm following the logic of the function correctly, it returns an array of values, except for the value passed to the function. When you test, you test for the returned value not to be the value passed. 

I think something more adequate would be to test for the values that are returned in the array. If there aren't any tasks in the document, it would return empty.

I will leave here some snippets that I worked on to give you an idea of what I refer to:

![image](https://user-images.githubusercontent.com/79114233/141231972-03f6fa62-32fc-48c7-8038-147d05d8d6f6.png)

_Note that these tests will give you errors if you try them, but you could try to fix issues in the logic in order to make them pass_